### PR TITLE
Added resume field in Application Page

### PR DESCRIPTION
**Pull request by IanWearsHat**
## Validation
- Form will prevent submission if:
  - File is not pdf
  - File is over 0.5 MB
## Feedback
- If any of the errors above occur, text will appear saying an error as occurred
- If successful, text will appear saying so
## Form Change
- Changed form to be multipart to allow a file to be sent

### CODE REVIEW

File: apps/site/src/app/apply/sections/Form/AgeInformation.tsx
Comment (samderanova) [inline apps/site/src/app/apply/sections/Form/AgeInformation.tsx:33]: Question: is there a reason why you changed these to underscores? If I remember correctly, @taesungh is working on writing the code to process the application, so you might want to sync with him before changing the names.

File: apps/site/src/app/apply/sections/Form/ResumeOutputFeedback.tsx
Comment (samderanova) [inline apps/site/src/app/apply/sections/Form/ResumeOutputFeedback.tsx:1]: Nitpick: We generally use TitleCase for interfaces.

File: apps/site/src/app/apply/sections/Form/ResumeOutputFeedback.tsx
Comment (samderanova) [inline apps/site/src/app/apply/sections/Form/ResumeOutputFeedback.tsx:6]: Suggestion: you can extract the properties out from the props like so: `OutputFeedback({errorMessage, resumePath}: OutputFeedbackProps)`

**Review by samderanova — CHANGES_REQUESTED** [review]
Great work on this! A few questions/nitpicks:

File: apps/site/src/app/apply/sections/Form/AgeInformation.tsx
Comment (taesungh) [reply apps/site/src/app/apply/sections/Form/AgeInformation.tsx:33]: Yes, I will be updating this field name along with others as part of #93. I'm not sure if we should wait for my PR or if I should try to work with these new changes.

Comment (samderanova) [reply apps/site/src/app/apply/sections/Form/AgeInformation.tsx:33]: If you haven't gotten too far with #93 yet (i.e. written the field names), we can merge this one first.

Comment (IanWearsHat) [reply apps/site/src/app/apply/sections/Form/AgeInformation.tsx:33]: I was testing to see if I could send the form in FastAPI (just a simple print statement), but for some reason having a hyphen in the parameters of the post request I was sending raised a compiler error, so I changed them all to underscores. 

Sorry, I should have thought about other people working with these names.

Comment (samderanova) [reply apps/site/src/app/apply/sections/Form/AgeInformation.tsx:33]: No, that makes sense. Python can't process hyphens in names so I think that's why you're receiving those errors.

**Review by samderanova — DISMISSED** [review]

File: apps/site/src/app/apply/sections/Form/Form.tsx
Comment (taesungh) [inline apps/site/src/app/apply/sections/Form/Form.tsx:42]: Chore: I think we still need the action endpoint here?

File: apps/site/src/app/apply/sections/Form/AgeInformation.tsx
Comment (taesungh) [reply apps/site/src/app/apply/sections/Form/AgeInformation.tsx:33]: No worries, this should probably have been noticed the first time around. I'm almost ready with the apply endpoint, but the file upload is also an important part I should test, so I can rebase after this is merged.

File: apps/site/src/app/apply/sections/Form/Form.tsx
Comment (IanWearsHat) [reply apps/site/src/app/apply/sections/Form/Form.tsx:42]: Oops, I thought I left it on merge

**Review by samderanova — APPROVED** [review]

**Review by taesungh — APPROVED** [review]
Approving to unblock #93, we can revisit the actual design: currently not a fan of the "Successfully uploaded" text when it actually hasn't been uploaded.

**Pull request merged**
